src/main/java/core/dao/DaoUtils.java

```diff
@@ -0,0 +1,54 @@
+package core.dao;
+
+import core.model.Car;
+import core.model.Driver;
+import core.model.Manufacturer;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DaoUtils {
```

Comment:
I would recommend to make the methods of this class private in the Dao classes where we use them

it would look better

Reply:
I asked Bohdan if I can do it this way, to avoid code duplication. And got an answer from him "Ok, let's try it. If Mentors will leave you comments on that, let them know I approved this approach."